client/components/navigation/Navbar.tsx

```diff
@@ -46,69 +53,172 @@ function Navbar() {
           </HStack>
         </Link>
 
-        <HStack spacing={{ base: 2, sm: 7 }}>
-        <Link
-          color="black"
-          p="0.45rem"
-          m="0 0.25rem"
-          cursor="pointer"
-          textDecoration="none"
-          _hover={{textDecoration: "none", bgColor:"#e2e8f0"}}
-          _focus={{textDecoration: "none", border:"none"}}
-          borderRadius="18px"
-          href="/gigs">Gigs</Link>
-        <Link
-          color="black"
-          p="0.45rem"
-          m="0 0.25rem"
-          cursor="pointer"
-          textDecoration="none"
-          borderRadius="18px"
-          _hover={{textDecoration: "none", bgColor:"#e2e8f0"}}
-          _focus={{textDecoration: "none", border:"none"}}
-          href="/jobs">Jobs</Link>
+        <Button bg="gray.300" display={{ sm: 'block', md: 'none' }} zIndex={11}>
+          {isHamburgerOpen ? (
+            <CloseIcon
+              h={6}
+              w={6}
+              onClick={() => setIsHamburgerOpen(!isHamburgerOpen)}
+            />
+          ) : (
+            <HamburgerIcon
+              h={7}
+              w={7}
+              onClick={() => setIsHamburgerOpen(!isHamburgerOpen)}
+            />
+          )}
+        </Button>
+        {isHamburgerOpen && (
+          <VStack
+            position="fixed"
+            zIndex={10}
+            top={0}
+            left={0}
+            h="100%"
+            w="100%"
+            display={{ sm: 'flex', md: 'none' }}
+            bg="white"
+            spacing={10}
+            fontSize="3xl"
+            pt={20}
+          >
+            <Link
+              color="black"
+              p="0.45rem"
+              m="0 0.25rem"
+              cursor="pointer"
+              textDecoration="none"
+              borderRadius="18px"
+              href="/gigs"
+            >
+              Gigs
+            </Link>
+            <Link
+              color="black"
+              p="0.45rem"
+              m="0 0.25rem"
+              cursor="pointer"
+              textDecoration="none"
+              borderRadius="18px"
+              href="/jobs"
+            >
+              Jobs
+            </Link>
```

Comment:
Since we're using Next.js, we need to be a bit fancy with how we render Chakra's `Link` component.

This is a good overview: https://jools.dev/using-nextjs-link-with-chakra-ui-link

Essentially we need to render an outer `NextLink` which accepts the `href` and passes that to the inner Chakra `Link`.

You can see the effect of not using `NextLink` by clicking one of these links. The entire page reloads when it should just render the next page (since it's a SPA).